source/iface/context.rst

```diff
@@ -59,86 +63,104 @@ or devices are supplied.
 
 .. list-table::
 
-  * - ``propList``
-    - See `sycl::property::context`_.
   * - ``asyncHandler``
     - Called to report asynchronous SYCL exceptions for this context
   * - ``dev``
     - Constructed context contains device
   * - ``deviceList``
     - Constructed context contains devices
   * - ``plt``
-    - Constructed context contains platform
-  * - ``clContext``
-    - Constructed context contains cl_context
-
+    - Constructed context instance using the ``plt`` parameter as the associated SYCL platform
```

Comment:
I'd say it like this:

> Constructed context contains all devices in platform ``plt``.

Reply:
Updated with : https://github.com/KhronosGroup/SYCL_Reference/pull/73/commits/567362eccf2163fc3dcb746bd4e2d38f6ecaa5ae